content/docs/state-and-lifecycle.md

```diff
@@ -68,23 +68,23 @@ ReactDOM.render(
 );
 ```
 
-To implement this, we need to add "state" to the `Clock` component.
+यसलाई कार्यान्वयन गर्न हामीले `Clock`कम्पोनेंटमा "state" थप्न आवश्यक छ।
 
-State is similar to props, but it is private and fully controlled by the component.
+State Props जस्तै छ तर यो निजी हुन्छ​ र पूरै कम्पोनेंटद्वारा नियन्त्रित हुन्छ​।
 
-## Converting a Function to a Class {#converting-a-function-to-a-class}
+## एउटा functionलाई classमा रूपान्तरण गर्ने तरिका{#converting-a-function-to-a-class}
 
-You can convert a function component like `Clock` to a class in five steps:
+तपाईं `Clock` जस्तो function कम्पोनेंटलाई classमा पाँच चरणमा रूपान्तरण गर्न सक्नुहुन्छ।
 
-1. Create an [ES6 class](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes), with the same name, that extends `React.Component`.
+१. [ES6 class](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes) सिर्जना गर्नुहोस् उही नामको साथ जसले `React.Component` विस्तार गर्दछ।
```

Comment:
What do you think about this? 
I reckon `extend` in this context is more techincal, so might not be useful translating? (only an opinion though)
```suggestion
१. उही नामको [ES6 class](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes) सिर्जना गर्नुहोस्  जसले `React.Component` extend गर्दछ।
```